packages/styles/index.html

```diff
@@ -32,5 +32,32 @@ <h1>Styles package playground</h1>
 
     <!-- Place your component markup here -->
     <button class="btn btn-primary">Primary button</button>
+
+    <div style="padding: 2rem">
```

Comment:
Isn't the segmented button replacing the button group? https://preview-3879--swisspost-design-system-next.netlify.app/?path=/docs/021d61aa-e039-4858-b4b9-b86a3e772811--docs 

If so it would be good not to change the markdown too much to ease project migration.

Reply:
Philipp said it would not be replacing the button group, only partially, this part 
![image](https://github.com/user-attachments/assets/05233d83-2612-4823-bc79-fdeb804c6129)